less/hotgraphic.less

```diff
@@ -143,6 +143,25 @@
         font-size:0.75em;
         line-height: 1em;
     }
+
+	.hotgraphic-widget .numbers {
```

Comment:
This selector's now broken unless you chain the two classes. I'd just remove `.hotgraphic-widget` as per original suggestion.